### PR TITLE
README: List required devel packages for header files

### PR DESCRIPTION
**Pull request by dghubble**
Clean checkout and build failed :(

```
$ ./build
Building ignition...
# github.com/coreos/ignition/internal/exec/util
gopath/src/github.com/coreos/ignition/internal/exec/util/blkid.c:15:25: fatal error: blkid/blkid.h: No such file or directory
 #include <blkid/blkid.h>
                         ^
compilation terminated.
```

### CODE REVIEW

**Comment by dghubble** [comment]
@crawford fixed message

I'm sure the RPM is right, and I think the deb is too, though I don't have a debian Go build environment handy.

```
apt-file list util-linux | grep blkid
util-linux: /sbin/blkid
util-linux: /usr/share/bash-completion/completions/blkid
util-linux: /usr/share/doc/util-linux/blkid.txt
util-linux: /usr/share/man/man8/blkid.8.gz
```

**Comment by crawford** [comment]
The title should read "readme: list required devel packages for header files". Please reference the commit history for particular files as a guide.

**Comment by dgonyeo** [comment]
I just hopped in a debian container to try this, and it looks like debian 8 still ships golang 1.3, which Ignition won't build under due to the lack of vendor directory support. Users could always manually install a newer version, but maybe it'd be easier on our end to drop mention of debian here?

**Comment by dghubble** [comment]
Or I can mention the Go version requirement as well? I'm guessing its 1.7+ based on the travis file?

I'd guess very few Go developers on Debian go about their day using 1.3.

**Comment by dgonyeo** [comment]
Saying 1.7+ is required would probably be good

**Comment by dghubble** [comment]
Mentioned now.

**Comment by dghubble** [comment]
@crawford mind dismissing your review if it looks ok now?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 
 Use the [bug tracker][issues] to report bugs.
 
+## Development
+
+Development requires a Go 1.7+ environment and the `blkid.h` headers.
+
+```sh
+# Debian/Ubuntu
+sudo apt-get install util-linux
```
Comment (lucab) [inline README.md]:
```
$ apt-file search blkid.h
libblkid-dev: /usr/include/blkid/blkid.h
```

That package also contains pkg-config settings.

Comment (dghubble) [inline README.md]: Switching to `libblkid-dev` then

**Pull request merged**